.vale/fixtures/RedHat/PascalCamelCase/testinvalid.adoc

```diff
@@ -0,0 +1,148 @@
+APIServer
+APIService
+BareMetalHost
+BrokerTemplateInstance
+BuildConfig
+CatalogSource
+CertificateSigningRequest
+CloudCredential
+ClusterAutoscaler
+ClusterCSIDriver
+ClusterNetwork
+ClusterOperator
+ClusterResourceQuota
+ClusterRole
+ClusterRoleBinding
+ClusterServiceVersion
+ClusterTask
+ClusterVersion
+ComponentStatus
+Config
+ConfigMap
+ConsoleCLIDownload
+ConsoleExternalLogLink
+ConsoleLink
+ConsoleNotification
+ConsoleYAMLSample
+ContainerRuntimeConfig
+ControllerConfig
+ControllerRevision
+CredentialsRequest
+CronJob
+CSIDriver
+CSINode
+CSISnapshotController
+CustomResourceDefinition
+DaemonSet
+Deployment
+DeploymentConfig
+DNSRecord
+EgressNetworkPolicy
+EndpointSlice
+Event
+FeatureGate
+FlowSchema
+Group
+HelmChartRepository
+HorizontalPodAutoscaler
+HostSubnet
+ImageContentSourcePolicy
+ImagePruner
+ImageSignature
+ImageStream
+ImageStreamImage
+ImageStreamImport
+ImageStreamMapping
+ImageStreamTag
+ImageTag
+Ingress
+IngressClass
+IngressController
+innerStopPod
+InstallPlan
+IPPool
+KubeAPIServer
+KubeControllerManager
+KubeletConfig
+KubeScheduler
+KubeStorageVersionMigrator
+Lease
+LimitRange
+LocalResourceAccessReview
+LocalSubjectAccessReview
+MachineAutoscaler
+MachineConfig
+MachineConfigPool
+MachineHealthCheck
+MachineSet
+MutatingWebhookConfiguration
+NetNamespace
+NetworkAttachmentDefinition
+NetworkPolicy
+newCustomerId
+Node
```

Comment:
Node Feature Discovery
Single Node OpenShift

Is the word Node going to trigger this rule? I'd leave it out if so